homework.md

```diff
@@ -4,110 +4,111 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
 
-1322
+12,240
 
-## Question 4 
+## Question 4
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+using CSS helps you build responsive, future-proof, and highly accessible websites.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
-
+h1 {
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+JavaScript is a text-based programming language used both on the client-side and server-side that allows you to make web pages interactive language used on the internet
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1-Remove a file from git without removing it from your file system. ...
+2-Undo local commits. ...
+3-Clean up local commits before pushing. ...
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+Iis producing a personal copy of someone else's projectt
```

Comment:
Just some typo corrections:

It is producing a personal copy of someone else's project

Reply:
thanks, Sharon for the corrections. 
Do you know how I can correct these mistakes?